integration/resources/templates/single/basic_application_sar_location_with_intrinsics.yaml

```diff
@@ -1,53 +1,52 @@
 Parameters:
-  SemanticVersion:
+  SarApplicationId:
     Type: String
-    Default: 1.0.2
 
 Mappings:
   SARApplication:
     us-east-1:
-      ApplicationId: arn:aws:serverlessrepo:us-east-1:077246666028:applications/hello-world-python3
+      SemanticVersion: 1.0.3
```

Comment:
is there a way to take rid of this configuration. We are adding the same value for all regions any way.

Reply:
I know it doesn't make any sense, but this is just to confirm that SAR integration can work with `Fn::FindInMap` and `Fn::Ref`. The previous content was re-using same SAR App ARN for all the regions.